app/viewmodels/TradeJourneyStatusesViewModel.scala

```diff
@@ -43,9 +44,13 @@ object TradeJourneyStatusesViewModel {
       if (optJourney.isEmpty) None else optJourney.head.completedState
     }
```

Comment:
Let's simplify this buildSummaryList, as a started let's move getStatus out of the buildSummaryList and make it norma def getStatus